source/uri-options/tests/connection-options.json

```diff
@@ -0,0 +1,147 @@
+{
+  "tests": [
+    {
+      "description": "Valid connection and timeout options are parsed correctly",
+      "uri": "mongodb://example.com/?appname=URI-OPTIONS-SPEC-TEST&connectTimeoutMS=20000&heartbeatFrequencyMS=5000&localThresholdMS=3000&maxIdleTimeMS=50000&replicaSet=uri-options-spec&retryWrites=true&serverSelectionTimeoutMS=15000&socketTimeoutMS=7500",
```

Comment:
This one is a challenge for drivers without a connection pool because of maxIdleTimeMS. Maybe put tests for things like that in a connection-pool-options.json and let PHP and Perl skip the whole file?

Reply:
Will do!